bleak/backends/scanner.py

```diff
@@ -41,3 +44,18 @@ async def set_scanning_filter(self, **kwargs):
     @abc.abstractmethod
     async def get_discovered_devices(self) -> List[BLEDevice]:
         raise NotImplementedError()
+
+    @classmethod
+    @abc.abstractmethod
+    async def find_specific_device(cls, device_identifier: str, timeout: float = 10.0) -> BLEDevice:
+        """A convenience method for obtaining a ``BLEDevice`` object specified by MAC address or (macOS) UUID address.
+
+        Args:
+            device_identifier (str): The MAC/UUID address of the Bluetooth peripheral sought.
+            timeout (float): Optional timeout to maximally wait for detection of specified peripheral. Defaults to 10.0 seconds.
```

Comment:
"maximally wait` sounds a bit odd to me as a native English speaker. Maybe say "time to scan before giving up" or something like that?

Reply:
Changed the wording on this.